en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts.md

```diff
@@ -0,0 +1,293 @@
+        ```
+
+    -   You can update [other configurations](https://github.com/wso2/kubernetes-microgateway/tree/v1.0.0.1/helm/choreo-connect/README.md) as required.
+
+3.  Save the `values.yaml` file.
+
+### Step 3 - Deploy Choreo Connect
+
+Once you have set up your Helm resources locally, follow the instructions given below to set up the deployment.
+
+1.  Open a terminal and navigate to the `<KUBERNETES_HOME>/helm/choreo-connect` directory.
+2.  Execute the command that is relevant to your Helm version.
+
+    !!! Tip
+        Be sure to replace `NAMESPACE` with the Kubernetes namespace in which your resources are deployed.
+
+    -   Using **Helm v2**
+
+        ```bash
+        helm install --name <RELEASE_NAME> wso2/choreo-connect --version 1.0.0-1 --namespace <NAMESPACE>
+        ```
+
+    -   Using **Helm v3**
+
+        ```bash
+        helm install <RELEASE_NAME> wso2/choreo-connect --version 1.0.0-1 --namespace <NAMESPACE> --create-namespace
+        ```
+
+#### Update configurations during deployment
+
+If required, you can set any of the deployment configurations at the time of running the deployment (instead of
+specifying them in the `values.yaml` file). See the examples given below.
+
+-   Setting the subscription username and password.
+
+    ```bash
+    --set wso2.subscription.username=<SUBSCRIPTION_USERNAME>
+    --set wso2.subscription.username=<SUBSCRIPTION_USERNAME>
+    ```
+
+-   Setting the Choreo Connect deployment mode. 
+
+    ```bash
+    --set wso2.deployment.mode=APIM_AS_CP
+    ```
+
+-   Use the Choreo Connect control plane configurations.
+
+    ```bash
+    --set wso2.apim.controlPlane.hostName=am.wso2.com
+    --set wso2.apim.controlPlane.serviceName=wso2am-single-node-am-service.apim
+    ```
+
+## Access the Choreo Connect deployment
+
+Follow the steps given below.
+
+1.  Get the external IP (`EXTERNAL-IP`) of the Ingress resources by listing down the Kubernetes Ingresses.
+
+    -   **API Manager - Control Plane**
+
+        ```bash
+        kubectl get ing -n apim
+        ```
+
+        Example:
+
+        ```bash
+        NAME                                       HOSTS                      ADDRESS        PORTS     AGE
+        wso2am-pattern-3-am-cp-ingress             am.wso2.com                <EXTERNAL-IP>  80, 443   3m
+        ```
+
+    -   **Choreo Connect**
+
+        ```bash
+        kubectl get ing -n <NAMESPACE>
+        ```
+
+        Example:
+
+        ```bash
+        NAME                                       HOSTS                      ADDRESS        PORTS     AGE
+        <RELEASE_NAME>-choreo-connect-router       gw.wso2.com                <EXTERNAL-IP>  80, 443   3m
+        ```
+
+2.  Add the above hosts in the `/etc/hosts` file as follows:
+
+    ```bash
+    <EXTERNAL-IP>   am.wso2.com
+    <EXTERNAL-IP>   gw.wso2.com
+    ```
+
+3.  Execute the following command to invoke health check services:
+
+    ```bash
+    curl -X GET "https://gw.wso2.com/health"
+    ```
```

Comment:
Let's remove this.

Reply:
fixed with https://github.com/wso2/docs-apim/pull/5214/commits/56e8d06f0eefdc6ceb6870305d7af0214650c154